### PR TITLE
Add Wagtail bakery

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/wagtail-bakery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (isuruf) [inline recipes/wagtail-bakery/meta.yaml]: Please add `noarch: python`

**Comment by dougalsutherland** [comment]
FYI, in the future you can make an empty commit with `git commit --allow-empty` rather than adding random blank lines. :)

Don't worry about the AppVeyor failure; see https://github.com/conda-forge/idna-feedstock/pull/5.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge. It is failing only for python 3.5 on windows due to some issues.

**Comment by scopatz** [comment]
@sannykr - I think it needs to pass CI before we merge.  Could you try restarting, please? Thanks!

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge if everything looks okay.

**Pull request merged**